static/app/components/events/interfaces/spans/spanDetail.tsx

```diff
@@ -309,16 +313,23 @@ function SpanDetail(props: Props) {
       <Alert type={getCumulativeAlertLevelFromErrors(relatedErrors)} system>
         <ErrorMessageTitle>
           {tn(
-            '%s error event occurred in this span.',
-            '%s error events occurred in this span.',
+            '%s error event or performance issue is associated with this span.',
+            '%s error events or performance issues are associated with this span.',
             relatedErrors.length
           )}
         </ErrorMessageTitle>
         <ErrorMessageContent>
           {visibleErrors.map(error => (
             <Fragment key={error.event_id}>
-              <ErrorDot level={error.level} />
-              <ErrorLevel>{error.level}</ErrorLevel>
+              {isErrorPerformanceError(error) ? (
+                <ErrorDot level="error" />
```

Comment:
Just to confirm - was the hardcoding of the level here intentional? Feels kind of like we should follow the underlying level here (even though I guess that would mean none(?) of our current issues show as errors...)

Reply:
Good 👁️ , but yes it was, we're going to go with "an issue is info outside of performance product, but is an error inside the performance product", I brought it up a couple weekly meetings ago. We're already showing `performance issues` as an error level on traces etc.